### ISSUE TITLE
Not generating audio. None at all. Generates graphs, but pitch silent wav file.

### ISSUE THREAD

**Issue by FlashlightET** [BenAAndrew/Voice-Cloning-App#69]
Neither the app nor colab produce audio, but they both do produce graphs
![image](https://user-images.githubusercontent.com/29938499/132406992-49b8ca46-ac13-4717-aeff-a9fd03bec54c.png)
File from program: https://cdn.discordapp.com/attachments/879773649714962553/884898833803399188/out.wav
File from colab (michael rosen model): https://cdn.discordapp.com/attachments/879773649714962553/884898840522670140/download.wav
![image](https://user-images.githubusercontent.com/29938499/132407122-980fa736-3398-4459-aa28-8b4c5e1b5a18.png)
![image](https://user-images.githubusercontent.com/29938499/132407177-23ad6308-4678-4b9a-9005-80bcd24624bc.png)
The colab seems to be generating actual audio though, since it shows a spectrogram, it just doesnt want to make it a wav file?

**Comment by BenAAndrew** [comment]
The issue here is due to the model quality. A typical alignment graph should look more like this.
![graph](https://user-images.githubusercontent.com/35925918/133260581-01416a1f-cf84-4dcc-9a28-cf9c4b3ea7ee.png)

If you cannot see a clear line forming, then the alignment is so poor that it won't be able to produce audible results.
Please post a message in the #help-wanted channel of our [discord](https://discord.gg/wQd7zKCWxT) if you'd like assistance identifying why the model is not good quality

**Comment by BenAAndrew** [comment]
To follow up on this issue, here are some common reasons why your voice may be poor: https://benaandrew.github.io/Voice-Cloning-App/training/#verifying-quality

**Comment by junqilu** [comment]
Thank you for developing this application and sorry for the bothering--I posted my question on discord but did not get any responses. I run into the same issue where the alignment graphs look legit for both the training and synthesis steps, but the generated audio is silent. I'm using the local .exe for the project and every step was done on my computer locally. 

According to your instructions on verifying the quality, I have 

- Train loss: about 0.08 < 0.5
- Attention score: about 0.77 > 0.3 
- Validation score: about 0.73 
- Alignment graph: looks like a clear line to me
![2023-01-11_131739](https://user-images.githubusercontent.com/40476820/211887673-05306a88-4913-4c7c-8017-5c71ca8fa441.jpg)

Also I am pretty sure that my training has passed at least 1000 epochs. For the vocoder, I tried both the one provided by your documentation page and the LJ_FT_T2_V3 from hifi-gan github page, but both gave silent audio but with a legit alignment graph. 
![2023-01-11_131826](https://user-images.githubusercontent.com/40476820/211888126-90a4c886-1eec-4d96-bca1-5efbaff9067c.jpg)

Could you provide some suggestions what might be an issue here? Really appreciate it!

**Comment by SirBitesalot** [comment]
What GPU are you using? I remember that some GPUs had some weird issue with some other ML stuff.

**Comment by junqilu** [comment]
I have 2 GPU on my computer--1 is Intel(R) Irus(R) Plus Graphics and the other is NVIDIA GeForce GTX 1660 Ti with Max-Q Design. I followed your video tutorial and installed the drive for NVIDIA GPU, so I always assume NVIDIA GPU is the one that the application is using.

**Comment by SirBitesalot** [comment]
its an issue with 1660/1660TI I think
https://github.com/pytorch/pytorch/issues/58123
You could try to use another version of CUDNN. Or try to run the app from source see if removing .half() can fix it like here.
https://github.com/NVIDIA/tacotron2/issues/475

**Comment by junqilu** [comment]
Thanks for the suggestions! I will definitely give it a shot. Does this mean my previous trained model is completely useless now and I should start fresh after making the changes as you suggested?

**Comment by SirBitesalot** [comment]
I think the model should be fine as only the inference uses .half() and it produces a normal graph.

**Comment by duckfromdiscord** [comment]
im getting silence with a normal graph and a 1660ti
